blog/models.py

```diff
@@ -0,0 +1,26 @@
+"""Mysite models."""
+from django.conf import settings
+from django.db import models
+from django.utils import timezone
+
+
+class Post(models.Model):
+    """Post model for blog app."""
+    
+    author = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+    )
+    title = models.CharField(max_length=200)
+    text = models.TextField()
+    created_date = models.DateTimeField(default=timezone.now)
+    published_date = models.DateTimeField(blank=True, null=True)
+
+    def publish(self):
+        """Mark the post as published."""
+        self.published_date = timezone.now()
+        self.save()
+
+    def __str__(self):
+        """Return post wich converted to string."""
```

Comment:
1) "w***h***ich" is misspelled. the verb is missing, mind your grammar.
2) Plz read the whole data model doc page but especially [this paragraph](https://docs.python.org/3/reference/datamodel.html#object.__str__)
3) You should describe the effect of having this method here which is just support for rendering `Post` instances into strings (by passing it to the `str()` function, usually), it's a part of the Python's string conversion protocol.

So you should just say that this returns a string representation of a Post object. Or, as I usually say, it renders Post as a string.